.github/workflows/pull-request-stake-pool.yml

```diff
@@ -99,11 +98,6 @@ jobs:
           key: node-${{ hashFiles('pnpm-lock.yaml') }}
           restore-keys: |
             node-
-      - name: Download programs
-        uses: actions/download-artifact@v2
-        with:
-          name: stake-pool-programs
-          path: target/deploy
```

Comment:
How come we don't need to download the programs anymore?

Reply:
The JS tests don't actually use the program, so I decided to simplify it for faster runs